include/timer.h

```diff
@@ -0,0 +1,60 @@
+/*************************************************************************
```

Comment:
Is this file used in the plugin?

Reply:
Yes. it is used in net_ib.cc.  so i have taken same changes